isaric/parsers/western-australia.toml

```diff
@@ -478,6 +472,24 @@
     { field = "overall_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "daily_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent on the first day of admission?" },
+    { field = "overall_exper_cmtype___1", values = { 1 = true } },
+    { field = "overall_exper_cmtype___2", values = { 1 = true } },
+    { field = "daily_exper_cmtype___1", values = { 1 = true } },
+    { field = "daily_exper_cmtype___2", values = { 1 = true } },
+    { field = "exper_cmtype___1", values = { 1 = true } },
+    { field = "exper_cmtype___2", values = { 1 = true } },
+  ]
```

Comment:
same comment as for RAPID re. the other antimalarials listed in *antimal_cmtype variables not yet mapped

Reply:
Thanks @sadiekelly  - I have added the extra variables